integration_test/test_resources/launcher-custom.yml

```diff
@@ -1,4 +1,5 @@
 configType: java
-configVersion: 1
-jvmOpts:
-  - '-Xmx1g'
+configVersion: 2
```

Comment:
yeah, that should be fairly easy to do. I think the exact implementation will depend on how we decide to handle generic executables vs. explicitly whitelisted executables